paypal/express/views.py

```diff
@@ -297,6 +297,13 @@ def build_submission(self, **kwargs):
 
         # Adding the tax to the total transaction amount to be sent to Paypal 
         self.txn.amount = self.txn.amount + basket.total_tax
+        
+        submission['shipping_method'].set_basket(basket)
```

Comment:
@monokrome  @tpgandhe  doing this because for some reason when we place the order the `shipping_method` is not assign a basket and returns a price of `0.00$` thus not saving the proper amount even though we charged the proper amount.